config/sewing-kit/index.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Package,
+  createComposedProjectPlugin,
+  createProjectTestPlugin,
+  createProjectBuildPlugin,
+} from '@sewing-kit/plugins';
+import {react} from '@sewing-kit/plugin-react';
+import {javascript} from '@sewing-kit/plugin-javascript';
+import {typescript} from '@sewing-kit/plugin-typescript';
+import {buildFlexibleOutputs} from '@sewing-kit/plugin-package-flexible-outputs';
+
+function addLegacyDecoratorSupport(config) {
+  return {
+    presets: config.presets,
+    plugins: (config.plugins || []).map(plugin => {
+      const pluginName = Array.isArray(plugin) ? plugin[0] : plugin;
+
+      if (pluginName.includes('@babel/plugin-proposal-decorators')) {
+        return [pluginName, {legacy: true}];
+      } else if (
+        pluginName.includes('@babel/plugin-proposal-class-properties')
+      ) {
+        return [pluginName, {loose: true}];
+      } else {
+        return plugin;
+      }
+    }),
+  };
+}
+
+export function quiltPackage({binaryOnly = true, jestEnv = 'jsdom'} = {}) {
+  return createComposedProjectPlugin<Package>('Quilt.Package', [
+    javascript(),
+    typescript(),
+    react(),
```

Comment:
We have some quilt packages that don't use any react (eg. `address-consts`). Do they still need the `react` plugin?

Reply:
Probably not, I can make that configurable via arguments instead 👍